### ISSUE TITLE
[Enchancement] Linux - NPCDefinitions could not be loaded, because Linux is case sensitive

### ISSUE THREAD

**Issue by MPNull** [Vult-R/Astraeus-Java-Framework#24]
I recommend a change:
```
package astraeus.io;

import java.io.FileReader;
import java.io.IOException;
import java.util.Arrays;

import com.google.gson.Gson;

import astraeus.game.model.entity.mob.npc.NpcDefinition;
import astraeus.util.GsonObjectParser;

public final class NpcDefinitionParser extends GsonObjectParser<NpcDefinition> {

  public NpcDefinitionParser() {
    super("./Data/npc/npc_definitions");
  }

  @Override
  public NpcDefinition[] deserialize(Gson gson, FileReader reader) throws IOException {
    return gson.fromJson(reader, NpcDefinition[].class);
  }

  @Override
  public void onRead(NpcDefinition[] array) throws IOException {
    Arrays.stream(array).forEach($it -> NpcDefinition.getDefinitions()[$it.getId()] = $it);
  }

}
```

To following:

```
package astraeus.io;

import java.io.FileReader;
import java.io.IOException;
import java.util.Arrays;

import com.google.gson.Gson;

import astraeus.game.model.entity.mob.npc.NpcDefinition;
import astraeus.util.GsonObjectParser;

public final class NpcDefinitionParser extends GsonObjectParser<NpcDefinition> {

  public NpcDefinitionParser() {
    super("./data/npc/npc_definitions");
  }

  @Override
  public NpcDefinition[] deserialize(Gson gson, FileReader reader) throws IOException {
    return gson.fromJson(reader, NpcDefinition[].class);
  }

  @Override
  public void onRead(NpcDefinition[] array) throws IOException {
    Arrays.stream(array).forEach($it -> NpcDefinition.getDefinitions()[$it.getId()] = $it);
  }

}
```

**Comment by Vult-R** [comment]
Fixed in commit #129. I searched through the project to make sure there wasn't any other classes that was using ./Data instead of ./data. PlayerDetails and PlayerContainer had to be fixed as well.

**Comment by Joshua-F** [comment]
You can link to commits by just posting the first 7 chars of the commit hash FYI.

6c3daf8
